legate/driver/args.py

```diff
@@ -188,7 +188,8 @@ def print_help(self, file: Optional[IO[str]] = None) -> None:
     dest="cprofile",
     action="store_true",
     required=False,
-    help="profile Python execution with the cprofile module",
+    help="profile Python execution with the cprofile module, "
+    "[legate-only, not supported with standard Python invocation]",
 )
 
```

Comment:
All of the arguments in the "Hardware binding" group in this file should also be marked as "legate-only".

Reply:
good catch, fixed now. Thank you.